mdata/notifierKafka/cfg.go

```diff
@@ -125,16 +124,9 @@ func ConfigProcess(instance string) {
 	partitionLogSize = make(map[int32]*stats.Gauge64)
 	partitionLag = make(map[int32]*stats.Gauge64)
 
-	// get the "newest" offset for all partitions.
 	// when booting up, we will delay consuming metrics until we have
```

Comment:
The rest of this comment needs to be cleaned up. All this loop is doing is initialising stats for each partition

Reply:
Done